### PR TITLE
cctp code added 

### PR DESCRIPTION
**Pull request by d4v1d03**

### CODE REVIEW

File: cctp/domains.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+    "0",
+    "1",
+    "2",
+    "3",
+    "6",
+    "7"
+]
```
Comment (pradyuman-verma) [inline cctp/domains.json:8]: No need for this file. It's not required

File: cctp/src/handlers/messageTransmitter/MintInfo.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import { mintTransactionsTable, ImintTransactionsTable } from "../../types/schema";
+import { attestationTable, IattestationTable } from "../../types/schema";
+
+/**
+* @dev Event::MessageReceived(address caller, uint32 sourceDomain, uint64 nonce, bytes32 sender, bytes messageBody)
+* @param context trigger object with contains {event: {caller ,sourceDomain ,nonce ,sender ,messageBody }, transaction, block, log}
+* @param bind init function for database wrapper methods
+*/
+export const MessageReceivedHandler = async (context: IEventContext, bind: IBind, secrets: ISecrets) => {
+// Implement your event handler logic for MessageReceived here
```
Comment (pradyuman-verma) [inline cctp/src/handlers/messageTransmitter/MintInfo.ts:11]: The code is not readable, can you use a linter or prettier

File: cctp/domains.json (hunk 1 above)
Comment (d4v1d03) [reply cctp/domains.json:8]: removed

File: cctp/src/handlers/messageTransmitter/MintInfo.ts (hunk 2 above)
Comment (d4v1d03) [reply cctp/src/handlers/messageTransmitter/MintInfo.ts]: done sir

**Comment by pradyuman-verma** [comment]
tests working 
image

Database
image

**Review by pradyuman-verma — APPROVED** [review]

**Pull request merged**